src/uiStrings.ts

```diff
@@ -125,4 +125,6 @@ export class UiStrings {
     static readonly RequestFailedWithStatusCode = vscode.l10n.t("Request failed with status code: {0}");
     static readonly DownloadDefinitionFileWithErrorMsg = vscode.l10n.t("Download API Center Definition File error: {0}");
     static readonly DatplaneAlreadyAdded = vscode.l10n.t("This Data Plane Runtime URL already added to Data View.");
+    static readonly SelectItemFromTreeView = vscode.l10n.t("Select from which tree view");
+    static readonly GetTreeView = vscode.l10n.t("Please Sign In to Azure to get management view or connect to data plane APIs");

```

Comment:
Please ask Sreek to help review all newly added strings.

`Sign In` -> `Sign in`

Better naming for `management view`

Reply:
update this sentence to "Please connect to Azure API Center Service first"